src/packages/shared-types/planType.ts

```diff
@@ -0,0 +1,16 @@
+export enum PlanType {
+  MED_SPA = "medicaid spa",
+  CHIP_SPA = "chip spa",
+}
```

Comment:
To kick off WW3, but Im not a fan of enums.  I think enums should be reserved for higher level domain(ish) use cases.  For a custom utility examples, like here, I lean towards type unions.

Reply:
This looks new, but it's just a name change from `actions.ts`. I'm indifferent about how we type _based_ on constant values, but I will always advocate for those constant values being imported instead of typed as individual strings. Could we infer value types from `PlanType` to keep a string union type in line with the enum? That could serve as a middle of the road pattern while we're working towards more modularized usage of constant string values.

The more we build utilities like `PlanCheck`, we'll likely end up reducing how often we use any given constant value elsewhere, and could probably eventually do away with the enum, but for now they're used largely to keep all our rogue strings around the codebase in check where needed.